src/app/containers/Headings/__snapshots__/index.test.jsx.snap

```diff
@@ -149,10 +151,8 @@ exports[`Headings with plain text headline should render h1 containing correct t
 exports[`Headings with plain text subheadline should render h2 containing correct text 1`] = `
 .c1 {
   color: #3F3F42;
-  font-family: ReithSans,Helvetica,Arial,sans-serif;
```

Comment:
Still has css removed from snapshots that isnt explained/replaced

Reply:
fixed!